sdformat_mjcf/sdformat_mjcf/converters/link.py

```diff
@@ -72,6 +72,7 @@ def add_link(body, link, parent_name="world", pose_resolver=su.pose_resolver):
         body.add("inertial",
                  mass=mass,
                  pos=su.vec3d_to_list(link.inertial().pose().pos()),
+                 euler=su.vec3d_to_list(link.inertial().pose().rot().euler()),
```

Comment:
This was left out intentionally. I believe Mujoco only allows either specifying the orientation of the inertial frame with a diagonal inertia matrix or fullinertia without specifying orientation. The orientation is accounted for in `link.inerital().moi()`. 

Tests pass, but when running the resulting file in Mujoco, I get `Error: error 'too many alternative definitions' in inertia alternative`. Perhaps we should start adding tests that actually run the simulator to check if the output mjcf file is valid.